src/lkl/setup.c

```diff
@@ -861,7 +861,8 @@ void lkl_mount_disks(
     if (!root)
         sgxlkl_fail("No root disk provided. Aborting...\n");
 
-    lkl_add_disks(root, mounts, num_mounts);
+    if (lkl_add_disks(root, mounts, num_mounts) != 0)
```

Comment:
`!= 0` is redundant here.  I normally prefer explicit checks here, but we're using this return value as a boolean and so C boolean semantics (0 is false, non-zero is true) are fine.  This test makes me wonder what the other return values are.

Reply:
Sorry @davidchisnall I replied your comment using VScode GitHub plugin, but apparently the comment didn't go through to github.

I saw in sgxlkl return code 0 is used as SUCCESS. And -1, 1, or -SOME_ERROR_CODE are used on failure. Which gave me an impression that it follows Unix program's convention (return 0 on success). I think if we use non-zero as success, then we need to check against specific error code. What do you think?

This PR was merged, if you insist we use non-zero as success, then I can make the change in my next PR.